### ISSUE TITLE
[Suggestion] Flag to disable iron doors opening on right click

### ISSUE THREAD

**Issue by tiggy26668** [Hidendra/LWC#629]
This would be extremely helpful for creating one way public doors on servers that work off of pressure plates or buttons.

At the moment there's no way, that I've found, to disable this feature for the normal player.

Currently I've tried every combination of /cprivate, /cpublic, /credstone, and /cautoclose,
but in any case either they can still just click the door to open, or can't open it at all even with pressure plates or buttons.

The only thing that does currently work is leaving the door unprotected, which has obvious flaws.

Basically this flag would allow the natural function of an unprotected iron door, with the benefit of it being registered protected door, preventing someone from claiming it.

This may be doable via the config file, I'm not sure, but that would change it for everyone, and not on a case by case basis as needed.

Thanks for the consideration and sorry if there's a way to do this that I've yet to figure out,
-Tiggy
